### PR TITLE
Report an issue form

### PR DESCRIPTION
**Pull request by JacobJaffe**
Adds `Report an Issue` to the `More` screen, which allows feedback submission to zendesk. Also updates our `.env` variables to be configured as dev / staging / release. 

Based off of https://github.com/Path-Check/gaen-mobile/pull/139, with changes to align with GPS use cases. 

image
image

### CODE REVIEW

File: app/helpers/getAppVersion.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { Platform } from 'react-native';
+
+const getAppVersion = (): string => {
+  const version = getVersion();
+  // Append "ALPHA" to our iOS builds that are 1.0.0, as we use
+  // a separate Alpha TestFlight that is always 1.0.0.
+  // On android we include "ALPHA" directly in the version name.
+  const isAlpha = version === '1.0.0';
+  const appVersion = `${
+    isAlpha && Platform.OS === 'ios' ? 'ALPHA ' : ''
```
Comment (vlasac) [inline app/helpers/getAppVersion.tsx]: Mind replacing this with `isPlatformIos` from `Util.js`?

File: app/views/ReportIssueForm.tsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    if (hasEmail && hasBody) {
+      setIsDisabled(false);
+    } else {
+      setIsDisabled(true);
+    }
+  };
+
+  useEffect(validate, [email, body]);
+
+  const isIOS = Platform.OS === 'ios';
```
Comment (vlasac) [inline app/views/ReportIssueForm.tsx]: Same as previous comment for `isPlatformIos`.

**Review by vlasac — APPROVED** [review]
Couple small nits here, but overall looks good. LGTM!

File: app/helpers/getAppVersion.tsx (hunk 1 above)
Comment (JacobJaffe) [reply app/helpers/getAppVersion.tsx]: Good call!

**Review by vlasac — APPROVED** [review]

**Pull request merged**